doc/examples/transform/plot_tps_deformation.py

```diff
@@ -0,0 +1,122 @@
+r"""
+=======================
+Thin Plate Spline (TPS)
+=======================
+
+TPS is the 2D generalization form of cubic spline [1]_. It is a two dimensional
+simulation of 1D cubic spline, which is the basic solution of Biharmonic Equation [2]_.
+According to [3]_, the basic solution of the biharmonic function was expanded.
+Given corresponding source and target points, TPS is used to compute
+a spatial deformation function for every point in the 2D plane or 3D volume.
+
+It should be noted that the commonly used in image processing
+Affine Transformation [4]_ can be understood as special variant of TPS.
+
+This script shows how to use the Thine Plate Spine Transformation for
+image warping.
+
+For further information on TPS Transformation, see:
+
+.. [1] Wikipedia, Spline interpolation
+       Shttps://en.wikipedia.org/wiki/Spline_interpolation
```

Comment:
```suggestion
       https://en.wikipedia.org/wiki/Spline_interpolation
```
Why pointing to this page instead of the [TPS's page](https://en.wikipedia.org/wiki/Thin_plate_spline)?

Reply:
Great spot. 
Upon reviewing, would [Cubic Spline](https://en.wikiversity.org/wiki/Cubic_Spline_Interpolation) be a better resource to point to?